src/test/resources/features/Scoring/Quiz/Quiz.feature

```diff
@@ -0,0 +1,139 @@
+Feature: Quiz
+
+  Background:
+    Given user prepare auth request
+    When user do login with email "admin@admin.com" and password "administratorfunctionapp"
+    Then user prepare quiz request with batchCode "futur3"
+    And user prepare question bank request
```

Comment:
I found it best to always call `And user hit logout endpoint` in each `Background`, since previous tests may alter the state of `authData` singleton